test/integration/ovs/ofctrl_test.go

```diff
@@ -1324,3 +1325,12 @@ func getCookieIDMask() (uint64, uint64) {
 	cookieMask := uint64(^uint16(0))<<48 | uint64(^uint8(0))<<40
 	return cookieID, cookieMask
 }
+
+func getFlowModificationMessages(flows []binding.Flow, operation binding.OFOperation) []*openflow15.FlowMod {
```

Comment:
can we just make the util function in `antrea.io/antrea/pkg/agent/openflow` public?

Reply:
updated.